pipelines/integration__previnity_negativacao/flow.py

```
@@ -0,0 +1,120 @@
+    save_materialization_datetime_redis,
+)
+from pipelines.integration__previnity_negativacao import constants
+from pipelines.integration__previnity_negativacao.tasks import (
+    get_previnity_credentials,
+    get_previnity_datetime_start,
+    prepare_previnity_payloads,
+)
+
+
+@flow(log_prints=True)
+async def integration__previnity_negativacao(  # noqa: PLR0913
```

Ruff (0.14.13)

Unused `noqa` directive (non-enabled: `PLR0913`)

Remove unused `noqa` directive

(RUF100)